src/language-js/index.js

```diff
@@ -6,64 +6,70 @@ const options = require("./options");
 const createLanguage = require("../utils/create-language");
 
 const languages = [
-  createLanguage(require("linguist-languages/data/javascript"), {
+  createLanguage(require("linguist-languages/data/JavaScript"), {
     override: {
       since: "0.0.0",
       parsers: ["babel", "flow"],
-      vscodeLanguageIds: ["javascript"]
-    },
-    extend: {
-      interpreters: ["nodejs"]
+      vscodeLanguageIds: ["javascript"],
+      interpreters: ["node", "nodejs"]
```

Comment:
Why?, original source contains all `interpreters`

Reply:
I was trying to make the result close as original as possible

Reply:
@fisker need to do it with understanding :smile: